pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -341,6 +341,7 @@ let
         # Telega has a server portion for it's network protocol
         telega = super.telega.overrideAttrs (old: {
           buildInputs = old.buildInputs ++ [ pkgs.tdlib ];
+          nativeBuildInputs = [ external.pkgconfig ];
```

Comment:
```suggestion
          nativeBuildInputs = [ external.pkg-config ];
```

Reply:
As far as I can see, both variants are valid. Other MELPA packages with native parts, for example pdf-tools, also use `pkgconfig`. Is there any difference between these packages?

Reply:
pkgconfig is an alias to pkg-config which we want to remove in the future and currently we are doing treewide PRs to remove the old alias. See eg #109595. Other packages should be moved away from in the near future and any new addition should not use the old alias.

Reply:
Thank you for explanation, fixed